examples/provider/bid/audience-network/src/audience-network-bid-delegate.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * @file Audience Network BidDelegate.
+ */
+import 'whatwg-fetch';
+
+const name = 'audience-network';
+
+/**
+ * Does this slot contain valid parameters?
+ * @param {Object} slot
+ * @returns {Boolean}
+ */
+const validateSlot = slot =>
+  typeof slot === 'object' &&
+  typeof slot.name === 'string' &&
+  typeof slot.elementId === 'string' &&
+  Array.isArray(slot.sizes) &&
+  slot.sizes.length > 0 &&
+  typeof slot.getParam === 'function' &&
```

Comment:
Ah, I completely forgot this object will have inherited properties, thanks for spotting this. I'll update to flatten the `sizes` property when accessed rather than attempt to clone the `slot`.